flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing. Since this is written in JAVA it should just run on an architecture, isn't it? "Write once, run everywhere" has been the slogan for Java for over 25 years now.

Reply:
There was no need/request so far by anybody for an ARM version, hence it's missing.

And clearly building Java applications inside flatpak is a PITA because maven can't download dependencies.
Once this is merged https://github.com/flatpak/flatpak-builder-tools/pull/253 , I'll probably change it to be built from source and this problem will go away.

Reply:
This still doesn't answer why it's exclusive to x86_64 when Java is supposed to be write once run everywhere.

Reply:
I don't want to get into arguments about Java honestly, I was simply trying to explain why currently there is only x86 version and not an ARM version.

But to add a bit more information,  yes the JAR runs everywhere. BUT, the desktop integration part that includes a JRE , creates convenient binaries and so on is platform specific and need to be rebuild for each platform.

Also i want to point out, this is  a side project for me I only spend time maintaining it because people find it useful. My daily work doesn't involve Java at all, so my knowledge of many things about Java are mediocre at best.